### PR TITLE
(RHEL-5991) Do not assign badness to filtered-out syscalls

### PR DESCRIPTION
**Pull request by dtardon**
Resolves: [#2196807](https://bugzilla.redhat.com/show_bug.cgi?id=2196807)

### CODE REVIEW

File: src/analyze/analyze-security.c [hunk 1]
```diff
@@ -1538,6 +1538,7 @@ static int assess(const struct security_info *info, Table *overview_table, Analy
                         return log_error_errno(r, "Failed to output table: %m");
         }
 
+        assert(weight_sum > 0);
```
Comment (msekletar) [inline src/analyze/analyze-security.c]: We already have assert on the correct place. This commit is not necessary.

File: src/analyze/analyze-security.c [hunk 2]
```diff
@@ -1536,7 +1536,6 @@ static int assess(const struct security_info *info, Table *overview_table, Analy
                         return log_error_errno(r, "Failed to output table: %m");
         }
 
-        assert(weight_sum > 0);
```
Comment (msekletar) [inline src/analyze/analyze-security.c]: Because commit which added this line is redundant this is also redundant.

**Review by msekletar — CHANGES_REQUESTED** [review]

File: src/analyze/analyze-security.c (hunk 1 above)
Comment (dtardon) [reply src/analyze/analyze-security.c]: Indeed.

File: src/analyze/analyze-security.c (hunk 2 above)
Comment (dtardon) [reply src/analyze/analyze-security.c]: Blah. Apparently I haven't checked carefully enough.

**Review by msekletar — APPROVED** [review]
LGTM

**Comment by jamacku** [comment]
@dtardon could you please rebase this PR, so CentOS CI would run again. Thank you.

**Pull request merged**